connaisseur/tests/test_admission_review.py

```diff
@@ -50,32 +60,34 @@ def adm_rev():
 
 
 @pytest.mark.parametrize(
-    "uid, allowed, patch, status_id, msg, review",
+    "uid, allowed, patch, status_id, msg, dm, review",
     [
-        (1, True, None, None, None, admission_review_plain),
-        (1, True, None, None, "Well hello there.", admission_review_msg),
+        (1, True, None, None, None, False, admission_review_plain),
+        (1, True, None, None, "Well hello there.", False, admission_review_msg),
         (
             1,
             True,
             [{"op": "add", "path": "/spec/replicas", "value": 3}],
             None,
-            None,
+            None, False,
```

Comment:
looking around, it seems like there should be one item per line here, for consistency. Consider using the Black code formatter.